backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -54,6 +58,34 @@ public static String parsePhoneNumber(String userSuppliedPhoneNumber) {
     }
   }
 
+  public static List<PhoneNumber> parsePhoneNumbers(List<PhoneNumberInput> phoneNumbers) {
+    if (phoneNumbers == null) {
+      return List.of();
+    }
+
+    return phoneNumbers.stream()
+        .map(
+            phoneNumberInput ->
+                new PhoneNumber(
+                    parsePhoneType(phoneNumberInput.getType()),
+                    parsePhoneNumber(phoneNumberInput.getNumber())))
+        .collect(Collectors.toList());
+  }
+
+  private static PhoneType parsePhoneType(String t) {
```

Comment:
We should be able to make the input layer do this by making PhoneNumberInput have an enum instead of a string for that field.